features/support/api_helpers.rb

```diff
@@ -52,15 +56,15 @@ def ensure_user_exists(user_details)
   creation_response = call_api(:post, "/users", payload: {
     users: user_details,
     updated_by: "functional tests",
-  })
+  }, safe_for_smoke_tests: true)
```

Comment:
Yes. It creates 1 specific user for each role and only if that user doesn't already exist in the environment. This is an alternative to preconfiguring environment users manually before running the tests, which just made running the tests locally harder.